docs/native-modules-ios.md

```diff
@@ -488,12 +490,29 @@ You will receive a warning if you expend resources unnecessarily by emitting an
     [self sendEventWithName:@"EventReminder" body:@{@"name": eventName}];
   }
 }
+@end
+```
+
+JavaScript code can subscribe to these events by creating a new `NativeEventEmitter` instance around your module.
+
+```jsx
+import { NativeEventEmitter, NativeModules } from 'react-native';
+const { CalendarManager } = NativeModules;
+
+const calendarManagerEmitter = new NativeEventEmitter(CalendarManager);
 
+const subscription = calendarManagerEmitter.addListener(
+  'EventReminder',
+  (reminder) => console.log(reminder.name)
+);
+// ...
+// Don't forget to unsubscribe, which is usually implemented in the componentWillUnmount lifecycle method.
+subscription.remove();
```

Comment:
The plain text paragraph/notes are preferred over the comment in example code. Can you convert this comment to the separate sentence and break the code of the example? 

Also `componentWillUnmount` refers to class components, and the suggested approach for functional components is missing.